style.css

```diff
@@ -442,6 +442,7 @@ image {
   color: #868a8f;
   width: 87%;
 }
+
 .form {
   background: #6070ff;
   background-image: url(contact-bg.png);
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design so make sure that you adjust the styling (the background image alignment of the form) to look exactly like the Figma design. :pray: 

| Your Design |Figma Design |
| :---:   | :---: | 
| ![image](https://github.com/mumbereh/my-portfolio/assets/92303891/600359b8-7bf6-4972-bb18-3bbb2504d05e) | ![image](https://github.com/mumbereh/my-portfolio/assets/92303891/e1df6261-0d04-47f9-a97c-9f387bcb042d)|